components/Storyblok/SbStoryListNav/SbStoryListNav.tsx

```diff
@@ -26,21 +26,23 @@ type SbStoryListNavType = {
 const StoryListContent = ({ blok: { links }, fullSlug }: SbStoryListNavType) => {
   return (
     <ul className={styles.list}>
-      {links.map((link) => {
-        const cached_url = link.link?.cached_url ? sbStripSlugURL(link.link.cached_url) : '';
-        const isCurrentPage = fullSlug === cached_url || (fullSlug === 'stories' && cached_url === 'stories/');
+      {links.map(({ _uid, label, link }) => {
+        const processedCachedUrl = sbStripSlugURL(link?.cached_url || '');
+        const processedFullSlug = sbStripSlugURL(fullSlug);
+        const isCurrentPage = processedFullSlug === processedCachedUrl;
+
```

Comment:
After I updated the sbStripSlugURL function, the current page highlight in the story filter page nav stop working, so I made some more tweaks to the function, and process both the full slug and the cached_url so that they're 100% in the same format before comparing them

Reply:
Aye, the forward and leading slashes get handled this way. 👍